edx_notifications/server/web/static/edx_notifications/templates/notification_pane.html

```diff
@@ -1,6 +1,35 @@
-<h3>Here are your notifications</h3>
 <ul class='edx-notifications-list'>
     <% _.each(user_notifications, function(user_notification){ %>
         <li class='edx-notifications-list-item'><%= user_notification.html %></li>
     <% }); %>
 </ul>
+<div class="container">
+    <div class="content">
+        <h2>Notifications</h2>
+        <div class="actions">
+            <ul class="notifications_list">
+                <li class="active"><a id="unread_notifications" href="#">View unread</a></li>
```

Comment:
We'll have to research how to support localization in Underscore/Backbone. I'm not sure, so I'll try to ask someone here. @andy-armstrong perhaps?